src/posts/getting-started-talkback.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Getting started with TalkBack on Android
+description: "A brief overview of getting started with TalkBack version 13.1 on Android."
+category: Assistive Technology
+author: Cooper Hollmaier
+date: 2023-02-12
+further_reading:
+  - title: "Android Accessibility Overview"
+    url: https://support.google.com/accessibility/android/answer/6006564?hl=en
+    source: Google
+  - title: "TalkBack: Hear Your Screen Out Loud"
+    url: https://support.google.com/accessibility/android/topic/3529932?hl=en&ref_topic=9078845
+    source: Google
+  - title: "TalkBack on Android Testing Steps"
+    url: https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/talkback-android.html
+    source: BBC
+tags:
+  - assistivetechnology
+---
+
+## Introduction to TalkBack
+
+TalkBack is assistive technology that helps users with visual disabilities access native mobile applications and the web by providing audible and haptic responses to touchscreen gestures. It is the default screen reader software included on Google touchscreen devices running the Android operating system.
+
+As of 2017, TalkBack is part of a family of software products in the Android Accessibility Suite including:
+- Accessibility Menu
+- Select to Speak
+- Switch Access
+
+The Android Accessibility Suite comes pre-installed on devices running Android 9 or greater.
+
+This article is an introduction to TalkBack version 13.1. We will walk you through the basics of enabling the software and navigating websites and applications with it.
+
+## Getting Started
+
+There are three ways to activate Talkback:
+1. From the settings menu: Accessibility > TalkBack > Toggle On
+2. On the physical device: Press and hold both volume keys for three seconds
+3. Via the Google Assistant: "Hey, Google. Turn on TalkBack"
```

Comment:
```suggestion
There are three ways to activate Talkback:

1. From the settings menu: **Accessibility** > **TalkBack** > **Toggle On**
2. On the physical device: Press and hold both volume keys for three seconds
3. Via the [Google Assistant](https://assistant.google.com/): "Hey, Google. Turn on TalkBack"
```

1. Newline for parsing.
2. Bold settings navigation steps (we do a similar treatment for a couple of other posts).
3. Link to Google Assistant for those who may be unfamiliar.